effects/effect.py

```diff
@@ -31,7 +26,7 @@ def __init__(self,
         self._activation_round: int
 
     def activate(self) -> None:
-        if self._status == EffectStatus.CREATED:
+        if (self._status == EffectStatus.CREATED):
```

Comment:
Are brackets necessary?

Reply:
Not necessary. Added for consistency with other code parts

Reply:
I always write without brackets. Now it is approved but we need to speak about one format for brackets, quotes etc. for each other.